wrench/reftests/scrolling/reftest.list

```diff
@@ -8,4 +8,4 @@
 == scroll-layer.yaml scroll-layer-ref.yaml
 == simple.yaml simple-ref.yaml
 == clip-and-scroll-property.yaml clip-and-scroll-property-ref.yaml
-== translate-nested.yaml translate-nested-ref.yaml
```

Comment:
This was a merge issue. Thanks for pointing it out. The line should be restored now.